src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class ColorSupplier {
+    private Random random = new Random();
```

Comment:
The Random instance here should be a constant field according to the Java style guide for constant names.